tests/integration_tests.rs

```diff
@@ -271,3 +271,79 @@ mod display_settings {
             );
     }
 }
+
+mod group {
+    use super::hexyl;
+
+    #[test]
+    fn group_2_bytes() {
+        hexyl()
+            .arg("ascii")
+            .arg("--color=never")
+            .arg("-g=2")
+            .assert()
+            .success()
+            .stdout(
+                "┌────────┬─────────────────────┬─────────────────────┬────────┬────────┐\n\
+             │00000000│ 3031 3233 3435 3637 ┊ 3839 6162 6364 650a │01234567┊89abcde_│\n\
+             └────────┴─────────────────────┴─────────────────────┴────────┴────────┘\n",
```

Comment:
is it possible to format this in the "right" way? or does `cargo fmt` destroy it?